src/components/CartDetailsTable.tsx

```diff
@@ -17,18 +17,19 @@ export default function CartDetailsTable(props: CartDetailsTableProps) {
     <div className="ordersCompleted">
       {props.cart.map((item) => {
         return (
-          <div className="orderCompletedDiv">
+          <div className="orderCompletedDiv" key={item.name}>
             <TokenCard
                 name={item.name}
                 uri={item.preview_uri}
                 width={300}
             />
-            {/* <tr key={item.name}>
-              {rowItem(item.config.size)}
-              {rowItem(item.config.frame)}
-              {rowItem(item.config.glass)}
-              {rowItem(item.config.space)}
-            </tr> */}
+
+            <div className="orderConfiguration">
+              <p>Size: {item.config.size} </p>
+              <p>Frame: {item.config.frame} </p>
+              <p>Glass Type: {item.config.glass} </p>
+              <p>Space Type: {item.config.space} </p>
+            </div>
           </div>
```

Comment:
not sure if you're missing a commit in this PR. but i think we still need the rowItems for the cart review screen

Reply:
"Need rowItems" ? The content is still there it's just not in the that div.

Reply:
yes, we use this component in the cart review screen. not sure if you intended this change for that screen too:

![Screenshot from 2021-02-18 10-01-37](https://user-images.githubusercontent.com/66155195/108400701-62d20780-71d0-11eb-969b-f5633dae8b40.png)

Reply:
the reason i ask if you're missing a commit is because i don't see any changes to the email component. unless it's already working? can you post a screenshot